Service Connector  Hub/func.py

```diff
@@ -1,49 +1,58 @@
 import io
-import oci
-import re
 import os
 import json
-import sys
-import requests
 import logging
-import time
-import gzip
-from fdk import response
 
-def process(body):
+import requests
+
+
+def process(body: dict) -> None:
+    data = body.get("data", {})
+    source = body.get("source")
+    time = body.get("time")
+    dd_source = "oracle_cloud"
```

Comment:
nit: consts should be all caps

Reply:
Or it can be initialized at module like: `_DD_SOURCE = "oracle_cloud"`